### ISSUE TITLE
@aaron-binary @binary-ex-machina  @wilburforce83 please help

### ISSUE THREAD

**Issue by saidforex26** [binary-com/binary-bot#1985]
how or where should I start building websites for bots like you guys have made

**Comment by wilburforce83** [comment]
Developer playground on binary.com website helped me a lot. We all use the binary-live-api it is a bit temperamental, and I know @binary-ex-machina and myself have both written our own workarounds to a number issues we've found with the API.

Definitely need either good JavaScript skills or a transferable skill. I think @binary-ex-machina is a skilled developer, whereas I am new to JS but I have a lot experience in other maths and engineering fields and was able to learn JS pretty quickly.

If you've never built a website from an IDE before i.e. not wordpress, I would do some YouTube tutorials on basic JS/HTML and CSS projects.

Good luck!

**Comment by saidforex26** [comment]
where did i find your youtube

**Comment by wilburforce83** [comment]
Hi, sorry, I don't understand, perhaps a translation problem? Could you rephrase? Thanks!

**Comment by saidforex26** [comment]
what is your youtube name. and can you make youtube tutoriar to use live API

**Comment by wilburforce83** [comment]
Ah, I understand. I am afraid I am way too busy at the moment to do a YouTube tutorial on the live API, though I might do one in a few months when work slows down a bit. I'd like to open source a better version of the CLI bot that is offered here.

I have built one that can backtest and only requires a 'strategy' .js file which will allow people with limited js skills the ability to build complex bots with all the benefits of JS

**Comment by saidforex26** [comment]
We are waiting for the development of your bot. Good luck.
